### ISSUE TITLE
Strange phenomenon on the KITTI dataset.

### ISSUE THREAD

**Issue by lianqing11** [WangYueFt/detr3d#26]
Dear Authors,

 Thanks for your wonderful work on 3D object detection. I am wondering that have you implemented detr3d on the KITTI dataset? I reimplemented it but observed strong overfitting on the KITTI dataset:

<img width="618" alt="image" src="https://user-images.githubusercontent.com/22816293/162116288-bf14094b-7267-4d76-a62f-bc19edbc8027.png">
<img width="1169" alt="image" src="https://user-images.githubusercontent.com/22816293/162116429-8b2f46d1-c956-4ae7-9a60-7de399055d87.png">
The first row denotes the validation accuracy and the second row denotes the training accuracy.

Best,
Qing LIAN

**Comment by a1600012888** [comment]
Hi, we have not tried. 

Your results is quite superising. Typically vision recognition models does not have such a strong overfitting.

**Comment by a1600012888** [comment]
Hi, Is this because the new parameterization of the orientation in newer version of mmdet3d
see: 
https://github.com/WangYueFt/detr3d/issues/31

**Comment by Hanqing-Sun** [comment]
@lianqing11 I'm facing the same problem on KITTI, have you solved it?
I've tried regenerating the gt database according to #31 , but with no luck.

**Comment by RashoAli** [comment]
@lianqing11 could you pleas provide the KITTI dataaset implimantation?

**Comment by reynerliu** [comment]
@lianqing11 could you please provide the new training config and KITTI dataset config? I tried to make it on my own, but training result is bad. after about 10 epoch,the grad_norm became nan and loss became 0. I tried to decrease learning rate, and the error still broke.I don't know how to solve it...I would be appreciated if you could give me some help!

**Comment by Archie945** [comment]
Can you share the kitti profile or training method, I'd really appreciate it!
请问能分享一下kitti的配置文件或者训练方法吗，实在感激不尽

**Comment by BatyrbekSM** [comment]
@lianqing11 Could you please provide KITTI implementation?
